setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 120
+exclude =
```

Comment:
We should also exclude .git, __pycache__, .idea, .vs and .vscode as this will make it a little quicker. We should really add them all to a .gitignore but we'll do that in another ticket.

Reply:
/venv/ is excluded in .gitignore by default and it won't be tested by flake8 when in a container. The line has been removed.

Reply:
People will also want to run the formatter on their machines (to actually make changes), if we add all the excludes it will run a little quicker for them and hopefully not ruin random files (like those in git)

Reply:
Ok. Excluded unwanted directory in setup.cfg according to https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/python/linting/flake8.html